src/main/java/sensors/ColorSensor.java

```diff
@@ -0,0 +1,74 @@
+package sensors;
+
+import com.revrobotics.ColorMatch;
+import com.revrobotics.ColorMatchResult;
+import com.revrobotics.ColorSensorV3;
+
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+/**
+ * code for color sensor detects color
```

Comment:
Which color sensor Brand and Name? How is it connected? Detects color of game piece